ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/util/CommandStoreUtil.java

```diff
@@ -40,7 +40,7 @@ public static void httpWaitForCommandSequenceNumber(
       final long seqNum = request.getCommandSequenceNumber().orElse(-1L);
       final String errorMsg = "Interrupted while waiting for command with the supplied "
           + "sequence number to execute. sequence number: " + seqNum
-          + ", request: " + request.getKsql();
+          + ", request: " + request.getMaskedKsql();
```

Comment:
Let's be defensive here and set the masked ksql if needed, so this doesn't NPE. I _think_ StreamedQueryResource (the other place besides KsqlResource) where this is called from goes through OldApiUtils but I'm not sure. Since we have plans to improve this in a bit anyway (https://github.com/confluentinc/ksql/issues/9252) let's be conservative in this PR and make our goal with this initial fix to not break anything.